meson_options.txt

```diff
@@ -47,3 +47,8 @@ option('log',
     type: 'boolean',
     value: false,
     description: 'Enable log message')
+
+option('static_library',
```

Comment:
I'd like to suggest a simpler option name here.
static_library => static

Reply:
I updated patch. Thank you